### PR TITLE
remove not supported verbose options

### PR DESCRIPTION
**Pull request by SpacePossum**
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4728#issuecomment-728897464

### CODE REVIEW

**Review by GrahamCampbell — APPROVED** [review]

File: doc/usage.rst [hunk 1]
```diff
@@ -31,10 +31,9 @@ The ``--verbose`` option will show the applied rules. When using the ``txt`` for
 
 NOTE: if there is an error like "errors reported during linting after fixing", you can use this to be even more verbose for debugging purpose
 
-* ``--verbose=0`` or no option: normal
-* ``--verbose``, ``--verbose=1``, ``-v``: verbose
-* ``--verbose=2``, ``-vv``: very verbose
-* ``--verbose=3``, ``-vvv``: debug
+* `-v`: verbose
+* `-vv`: very verbose
+* `-vvv`: debug
```
Comment (keradus) [inline doc/usage.rst:36]: what is the change?
`-vvv` works same as `--verbose=3` and both of them are provided out of the box by Symfony Console component.

what is the purpose of dropping one way of using it?

the fact that we don't print anything extra in debug verbosity is still there

Comment (SpacePossum) [reply doc/usage.rst:36]: >-vvv works same as --verbose=3

I doesn't, `--verbose=3` doesn't work:

```
possum@aquarium:/home/possum/work/PHP-CS-Fixer$ php php-cs-fixer fix --verbose=3

In ArgvInput.php line 204:
                                                          
  [Symfony\Component\Console\Exception\RuntimeException]  
  The "--verbose" option does not accept a value.         
                                                          

Exception trace:
  at /home/possum/work/PHP-CS-Fixer/vendor/symfony/console/Input/ArgvInput.php:204
 Symfony\Component\Console\Input\ArgvInput->addLongOption() at /home/possum/work/PHP-CS-Fixer/vendor/symfony/console/Input/ArgvInput.php:140
 Symfony\Component\Console\Input\ArgvInput->parseLongOption() at /home/possum/work/PHP-CS-Fixer/vendor/symfony/console/Input/ArgvInput.php:76
 Symfony\Component\Console\Input\ArgvInput->parse() at /home/possum/work/PHP-CS-Fixer/vendor/symfony/console/Input/Input.php:55
 Symfony\Component\Console\Input\Input->bind() at /home/possum/work/PHP-CS-Fixer/vendor/symfony/console/Command/Command.php:217
 Symfony\Component\Console\Command\Command->run() at /home/possum/work/PHP-CS-Fixer/vendor/symfony/console/Application.php:920
 Symfony\Component\Console\Application->doRunCommand() at /home/possum/work/PHP-CS-Fixer/vendor/symfony/console/Application.php:266
 Symfony\Component\Console\Application->doRun() at /home/possum/work/PHP-CS-Fixer/src/Console/Application.php:90
 PhpCsFixer\Console\Application->doRun() at /home/possum/work/PHP-CS-Fixer/vendor/symfony/console/Application.php:142
 Symfony\Component\Console\Application->run() at /home/possum/work/PHP-CS-Fixer/php-cs-fixer:98

fix [--path-mode PATH-MODE] [--allow-risky ALLOW-RISKY] [--config CONFIG] [--dry-run] [--rules RULES] [--using-cache USING-CACHE] [--cache-file CACHE-FILE] [--diff] [--diff-format DIFF-FORMAT] [--format FORMAT] [--stop-on-violation] [--show-progress SHOW-PROGRESS] [--] [<path>...]
```

**Comment by GrahamCampbell** [comment]
🚢

File: doc/usage.rst (hunk 1 above)
Comment (keradus) [reply doc/usage.rst:36]: haha, i was trying on `help`, @SpacePossum 
```
php php-cs-fixer help --verbose=3
```

which doesn't cause app crash, but now i see it's simply ignored (one can put `--sdafasdf=asdfafd` for `help` and it does not complain)

:+1:

Comment (SpacePossum) [reply doc/usage.rst:36]: good catch though as I haven't try the the `help` command 👍   thanks for checking!

**Pull request merged**